PR_REVIEW_CHECKLIST.md

```diff
@@ -0,0 +1,98 @@
+# PR review checklist
+
+This document lists specific checks that should be done when reviewing a PR. This means these items should be checked by:
+1. Every contributor submitting a Pull Request
+2. Every C+ (Contributor Plus) team member reviewing a PR
+3. Every Expensify employee submitting / reviewing a PR
+
+This is a living document, so we will be updating this from time to time. Feel free to suggest additions or changes in the #expensify-open-source Slack channel.
```

Comment:
I thought it would be useful to at least clarify that the process will be changing in the next few weeks / months as we get everything nailed down. I agree we don't necessarily want to advertise giving suggestions, we can do that in C+ channel if we want their ideas